domains/parksung.is-a-fullstack.dev.json

```diff
@@ -0,0 +1,23 @@
+{
+    "description": "Project Description",
+    "domain": "is-not-a.dev",
+    "subdomain": "parksung",
+
+    "owner": {
+        "repo": "https://github.com/username/repo",
+        "email": "youmansingmar@gmail.com"
+    },
+
+    "record": {
+        "A": ["15.235.136.239"],
+        "MX": ["mx1.example.com", "mx2.example.com"],
+        "TXT": ["v=spf1 a mx ip4:15.235.136.239 ~all"],
+        "TXT": ["v=DMARC1; p=none;"],
+        "TXT": ["v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCdMEKYRsrj8NCLHQJm/MgkYhuK
+5qatNZm2YJeySWkQ0Jj+u4JgbMtS91ZfiKOpKKD5NuJEDmqFgNcOudLTz3oSkq7s
+wdEGoNerXepVZ6AHuxYapGvBKkz2ALQh/lsggIiLG0GJcTDPXj3BX744HYSl2fZe
+jaB13MIYQZD1JqginQIDAQAB;"],
+    },
```

Comment:
TXT records should be all in an array, not multiple TXT properties which is invalid.